gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +0,0 @@
-#Mon Mar 07 21:06:45 EET 2016
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.12-all.zip
```

Comment:
You don't get the pint, if I use wrapper:
  1. I don't need to install gradle at my system at all
  2. Build didn't depend on version of gradle installed locally, each build exactly specify which version of gradle is to be used

That's best practices